cli/BUILD

```diff
@@ -33,17 +33,24 @@ nodejs_binary(
     entry_point = "df/cli/index.js",
 )
 
-load("@build_bazel_rules_nodejs//:defs.bzl", "npm_package")
+load("//tools/common:copy.bzl", "copy_file")
 
-npm_package(
-    name = "package",
-    srcs = ["package.json", "README.md"],
-    deps = [":cli"],
+copy_file(
```

Comment:
do we need this? i understood bazel to support per-file dependencies natively, i.e. that the dependent BUILD rule could already just depend on a specific file.

Reply:
It kind of does, but the npm_package rules get upset when the file isn't in the same directory as everything else and I haven't found another workaround :/